xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -208,12 +208,18 @@
       #set($discard = $allMatchingParams.addAll($filterParams))
     #end
   #end
+  #if ($request.debug) ##Mode Debug
```

Comment:
There is already a debug output at [line 425](https://github.com/Pbast/xwiki-platform/blob/2c1f0b5d85d25bdce3ffcbba111cd7f96f79516f/xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml#L425), why don't you extend that one?